pandapipes/component_models/circulation_pump_mass_component.py

```diff
@@ -46,24 +66,12 @@ def create_pit_node_entries(cls, net, node_pit):
         mf = np.nan_to_num(circ_pump.mdot_kg_per_s.values)
         mass_flow_loads = mf * circ_pump.in_service.values
         juncts, loads_sum = _sum_by_group(get_net_option(net, "use_numba"),
-                                          circ_pump.to_junction.values, mass_flow_loads)
+                                          circ_pump.return_junction.values, mass_flow_loads)
         junction_idx_lookups = get_lookup(net, "node", "index")[
             cls.get_connected_node_type().table_name()]
         index = junction_idx_lookups[juncts]
         node_pit[index, LOAD] += loads_sum
```

Comment:
If I compare this to the other circulation pump I have the feeling, that this needs to be adapted. Following things as example should be revised:

1. Why is it not mdot_flow_kg_per_s
2. As this is branch_component now, the mass flow should not be part of the node_pit, as in fact there is no load at the return_junction.

Reply:
For the first comment, I understand that you would prefer to change the column name? That would be fine to me, just requires another API change, like for flow / return junction.

With respect to the node_pit, I believe that this is the only way to implement the circulation pump in our current Jacobi-matrix setup. I believe that we will definitely end up with a new structure in future, where each component can feed its own model right into the jacobi matrix. But this would be a long way to go still, and maybe it should be a step to take afterwards.

Reply:
1. I would do it due to consistency reasons.
2. Agree. First things first.